src/test/java/com/adyen/WebhookTest.java

```diff
@@ -425,4 +424,260 @@ public void testManagementWebhookMerchantCreatedParsing() {
+                "              \"lastName\": \"Lastname\"\n" +
+                "            },\n" +
+                "            \"personalData\": {\n" +
+                "              \"dateOfBirth\": \"1981-02-27\",\n" +
+                "              \"documentData\": [\n" +
+                "                {\n" +
+                "                  \"expirationDate\": \"2030-12-31\",\n" +
+                "                  \"issuerCountry\": \"NL\",\n" +
+                "                  \"issuerState\": \"NH\",\n" +
+                "                  \"number\": \"ID#123456\",\n" +
+                "                  \"type\": \"ID\"\n" +
+                "                }\n" +
+                "              ],\n" +
+                "              \"nationality\": \"NL\"\n" +
+                "            },\n" +
+                "            \"phoneNumber\": {\n" +
+                "              \"phoneCountryCode\": \"NL\",\n" +
+                "              \"phoneNumber\": \"858888138\",\n" +
+                "              \"phoneType\": \"Landline\"\n" +
+                "            },\n" +
+                "            \"shareholderCode\": \"SH00001\",\n" +
+                "            \"shareholderReference\": \"SHREF#0000001\",\n" +
+                "            \"webAddress\": \"https://www.adyen.com/\"\n" +
+                "          }\n" +
+                "        ],\n" +
+                "        \"taxId\": \"TaxID#1234\"\n" +
+                "      },\n" +
+                "      \"email\": \"contact@adyen.com\",\n" +
+                "      \"fullPhoneNumber\": \"+31858888138\",\n" +
+                "      \"individualDetails\": {\n" +
+                "        \"name\": {\n" +
+                "          \"firstName\": \"Firstname\",\n" +
+                "          \"gender\": \"UNKNOWN\",\n" +
+                "          \"infix\": \"infix\",\n" +
+                "          \"lastName\": \"Lastname\"\n" +
+                "        },\n" +
+                "        \"personalData\": {\n" +
+                "          \"dateOfBirth\": \"1981-02-27\",\n" +
+                "          \"documentData\": [\n" +
+                "            {\n" +
+                "              \"expirationDate\": \"2030-12-31\",\n" +
+                "              \"issuerCountry\": \"NL\",\n" +
+                "              \"issuerState\": \"NH\",\n" +
+                "              \"number\": \"ID#123456\",\n" +
+                "              \"type\": \"ID\"\n" +
+                "            }\n" +
+                "          ],\n" +
+                "          \"nationality\": \"NL\"\n" +
+                "        }\n" +
+                "      },\n" +
+                "      \"merchantCategoryCode\": \"1212\",\n" +
+                "      \"metadata\": {\n" +
+                "        \"MetaKey\": \"MetaValue\"\n" +
+                "      },\n" +
+                "      \"payoutInstrumentTokens\": [\n" +
```

Comment:
This field is not present in the model, is the example wrong or are the specs wrong?
```suggestion
                "      \"payoutInstrumentTokens\": [\n" +
```

Reply:
example on the specs is wrong, resolving this.